docs/semgrep-supply-chain/license-scanning.md

```diff
@@ -0,0 +1,115 @@
+
+![Screenshot of Semgrep Supply Chain Dependencies tab with licenses listed](/img/sc-license-scanning.png)
+*Figure 1*. Screenshot of Semgrep Supply Chain Dependencies tab with licenses listed.
+
+:::info Prerequisites
+* License scanning can accessed through Semgrep Cloud Platform (SCP). [Create an account](/semgrep-code/getting-started/#signing-in-to-semgrep-cloud-platform) to view and manage license workflows.
+* To detect licenses, you must [add or onboard a project](/semgrep-code/getting-started/#option-b-adding-a-repository-from-github-gitlab-or-bitbucket) (repository) to Semgrep Cloud Platform for scanning.
+:::
+
+## Viewing licenses
+
+To view a package's license:
+
+1. [Sign in to Semgrep Cloud Platform](https://semgrep.dev/login).
+2. Click **[Supply Chain](https://semgrep.dev/orgs/-/supply-chain)** > **Dependencies**. By default licenses are listed in the row of their respective package.
+3. If you don't see any licenses:
+    1. Click the Supply Chain **Settings** on the header menu. These settings are specific to Semgrep Supply Chain.
+    ![Screenshot of Semgrep Supply Chain Settings tab](/img/sc-settings.png) *Figure 2*. Screenshot of Semgrep Supply Chain Settings tab.
+    2. Click <i class="fa-solid fa-toggle-large-on"></i> **Dependency search** if it is not already enabled.
+    3. Click <i class="fa-solid fa-toggle-large-on"></i> **Show licenses** if it is not already enabled.
+4. Optional: Search for a specific license by entering its name in the search box.
+
+## Blocking, commenting, or allowing licenses 
+
+This section provides guides on blocking or allowing packages in CI pipelines based on the license of a package.
+
+### Types of license permissions
+
+Licenses in Semgrep are assigned the following types of permissions:
+
+<dl>
+<dt>Allow</dt>
+<dd>Packages with licenses assigned this type of permission are allowed for use in the codebase.</dd>
+<dt>Comment</dt>
+<dd>Packages with licenses assigned this type of permission are allowed for use in the codebase and the PR author receives a comment.</dd>
+<dt>Block</dt>
+<dd>Packages with licenses assigned this type of permission are not allowed into the codebase. A comment is added to the PR or MR.</dd>
+</dl>
+
+To change the permissions of packages based on the license:
+
+1. From the Supply Chain pack, click **Settings** on the header menu. 
+2. Click <i class="fa-solid fa-toggle-large-on"></i> **Notifications and restrictions** if it is not already enabled.
+3. Browse the available licenses. 
+4. Click the permission (Allow, Comment, or Block) you want to set the license to.
+5. Optional: Block entire categories of licenses by clicking on the **Set all to** drop-down box next to the license category.
+
+### License categories
+
+The following license categories and licenses are identified by Semgrep Supply Chain:
+
+#### Weak copyleft licenses
+
```

Comment:
@andy-r2c I know we have a description for this, but I'm not seeing anything come out in the tooltips when testing. I wanted to sync my description with the tooltip in the app. Do you have a mock that shows the tooltip?

Reply:
<img width="975" alt="Screen Shot 2023-06-05 at 5 27 50 PM" src="https://github.com/returntocorp/semgrep-docs/assets/100232702/8c8b3984-53d9-40d2-b389-26a318dbb18b">

Reply:
try [matthew.dev.semgrep.dev](https://matthew.dev.semgrep.dev/orgs/-/supply-chain/settings) again?

Reply:
Thanks Andy. Resolving.